File: Class/1013/Part354ToggleEvent/app/src/main/java/com/example/part3_5_4_toggleevent/MainActivity.java
package com.example.part3_5_4_toggleevent;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;
import android.widget.ToggleButton;

public class MainActivity extends AppCompatActivity {

    ToggleButton objbtn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        objbtn=(ToggleButton)findViewById(R.id.toggleButton1);
        objbtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                if (objbtn.isChecked()){
                    Toast.makeText(MainActivity.this,"ON",Toast.LENGTH_SHORT).show();
                }else{
                    Toast.makeText(MainActivity.this,"OFF",Toast.LENGTH_SHORT).show();
                }
            }
        });
    }
}

File: Kotlin/week1/ImageViewkt/app/src/main/java/com/example/imageviewkt/MainActivity.kt
package com.example.imageviewkt

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)



        btn_toast.setOnClickListener{

            iv_profile.setImageResource(R.drawable.android2) //imageview의 새로운 이미지 등록
            Toast.makeText(this@MainActivity, "버튼이 클릭 되었습니다.", Toast.LENGTH_SHORT).show()
        }
    }
}